packages/react-components/components/SelectionOption.tsx

```diff
@@ -8,7 +8,6 @@ interface Props {
   word: string
   selected: boolean
   onSelectAnswer: (word: string, data: any) => void
-  key: string
```

Comment:
Can you confirm this doesn't affect the other consumers of this component (the language selection screen and the backup phrase quiz)?

Reply:
I checked it doesn't affect other consumers of this component 👍 

The `key` property is implicitly handled by React. The explicit type addition here wasn't necessary, I guess the intention was to force consumers of `SelectionOption` to pass it? Passing or not `key` depends on the usage. This PR showed a new use case which doesn't require it (`FlatList` items are already wrapped by keyed component).